### PR TITLE
update event detail mentions

### PR DESCRIPTION
**Pull request by lizokm**
## DESCRIBE YOUR PR
Deleted all mentions of the "Event Detail" page, which has been replaced by the Trace View page.

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [ ] Urgent deadline (GA date, etc.): 
- [X] Other deadline: June 5 
- [ ] None: Not urgent, can wait up to 1 week+

## SLA

- Teamwork makes the dream work, so please add a reviewer to your PRs.
- Please give the docs team up to 1 week to review your PR unless you've added an urgent due date to it.
Thanks in advance for your help!

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## LEGAL BOILERPLATE

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## EXTRA RESOURCES

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

File: docs/product/performance/web-vitals/index.mdx [hunk 1]
```diff
@@ -42,7 +42,7 @@ The **Web Vitals** page only displays data from your application's initial page
 
 ## Page Overview
 
-From the table in **Web Vitals** or a Web Vital Summary panel, click on a page to open its **Page Overview**. Here, you can see a summary of that single page's Web Vitals. In **Page Overview**, you can further drill down to a specific page load sample [Event](/product/sentry-basics/concepts/tracing/event-detail/), [Interaction](/product/performance/web-vitals/web-vitals-concepts/#interaction-to-next-paint-inp), [Replay](/product/session-replay/), or [Profile](/product/profiling/).
+From the table in **Web Vitals** or a Web Vital Summary panel, click on a page to open its **Page Overview**. Here, you can see a summary of that single page's Web Vitals. In **Page Overview**, you can further drill down to a specific page load sample [Event](/product/sentry-basics/concepts/tracing/concepts/key-terms/tracing/trace-view/), [Interaction](/product/performance/web-vitals/web-vitals-concepts/#interaction-to-next-paint-inp), [Replay](/product/session-replay/), or [Profile](/product/profiling/).
```
Comment (lizokm) [inline docs/product/performance/web-vitals/index.mdx:45]:
```suggestion
From the table in **Web Vitals** or a Web Vital Summary panel, click on a page to open its **Page Overview**. Here, you can see a summary of that single page's Web Vitals. In **Page Overview**, you can further drill down to a specific page load sample [Event](/concepts/key-terms/tracing/trace-view/), [Interaction](/product/performance/web-vitals/web-vitals-concepts/#interaction-to-next-paint-inp), [Replay](/product/session-replay/), or [Profile](/product/profiling/).
```

File: docs/concepts/key-terms/tracing/distributed-tracing.mdx [hunk 2]
```diff
@@ -243,4 +243,4 @@ Put simply: as a result of this head-based approach, where the decision is made
 
 ## Viewing Trace Data
 
-Through [Performance](/product/performance/) and [Discover](/product/discover-queries/), you can view trace data in the [event details](/product/sentry-basics/concepts/tracing/event-detail/).
+Through [Performance](/product/performance/) and [Discover](/product/discover-queries/), you can view trace data in the [trace view](/concepts/key-terms/tracing/trace-view/).
```
Comment (vivianyentran) [inline docs/concepts/key-terms/tracing/distributed-tracing.mdx:246]: We'll need to add the trace explorer page when it launches. Not sure if there are other entry points to the trace view, maybe also the insights modules and profiles?

File: docs/product/performance/mobile-vitals/app-starts.mdx [hunk 3]
```diff
@@ -104,4 +104,4 @@ Sentry automatically identifies sample events to help you investigate performanc
 
 You can use the sample list to drill down to and compare fast, average, and slow events of interest within a given screen.
 
-Clicking on a sample event will take you into either the query’s span details within the span waterfall of the **Event Details** page or the profile icon to see the [flamegraph](/product/profiling/flame-charts-graphs/) for the event. To get different event samples, click the "Try Different Samples" button.
+Clicking on a sample event will take you into either the query’s span details within the span waterfall of the [Trace View page](/concepts/key-terms/tracing/trace-view/#product-walkthrough-trace-view-page) or the profile icon to see the [flamegraph](/product/profiling/flame-charts-graphs/) for the event. To get different event samples, click the "Try Different Samples" button.
```
Comment (vivianyentran) [inline docs/product/performance/mobile-vitals/app-starts.mdx:107]: Looks like the page has been renamed to just "Trace" not "Trace View". Maybe something we want to update?

File: docs/product/performance/resources.mdx [hunk 4]
```diff
@@ -164,6 +164,6 @@ These same events are also represented as triangles shown in the average duratio
 
 You can generate a new list of random sample events by clicking the "Try Different Samples" button at the bottom.
 
-From the sample list, you can drill down to specific good, average, or bad examples of a given resource within a given page. Click on an event ID to drill into the resource's span details within the span waterfall of the **Event Details** page.
+From the sample list, you can drill down to specific good, average, or bad examples of a given resource within a given page. Click on an event ID to drill into the resource's span details within the span waterfall of the [Trace view](/concepts/key-terms/tracing/trace-view/#product-walkthrough-trace-view-page) page.
```
Comment (vivianyentran) [inline docs/product/performance/resources.mdx:167]:
```suggestion
From the sample list, you can drill down to specific good, average, or bad examples of a given resource within a given page. Click on an event ID to drill into the resource's span details within the span waterfall of the [Trace View](/concepts/key-terms/tracing/trace-view/#product-walkthrough-trace-view-page) page.
```

File: docs/product/performance/transaction-summary.mdx [hunk 5]
```diff
@@ -44,7 +44,7 @@ While you can use the search field to narrow down your events, you can also filt
 
 ### Transaction Table
 
-On initial load, the table displays slow occurrences of the transaction along with the Event ID, User, [Operations Breakdown](/product/sentry-basics/tracing/event-detail/#operations-breakdown), Total Transaction Duration, and Timestamp of the event. Click on the Event ID to open the [event detail](/product/sentry-basics/tracing/event-detail/) or the Trace ID to view the [full trace](/product/sentry-basics/tracing/trace-view). Other sets of transactions you can browse through are below.
+On initial load, the table displays slow occurrences of the transaction along with the Event ID, User, Operations Duration, Total Transaction Duration, and Timestamp of the event. Click on the Event ID to open the [trace view](/concepts/key-terms/tracing/trace-view/#product-walkthrough-trace-view-page) or the Trace ID to view the [full trace](/product/sentry-basics/tracing/trace-view). Other sets of transactions you can browse through are below.
```
Comment (vivianyentran) [inline docs/product/performance/transaction-summary.mdx:47]:
```suggestion
On initial load, the table displays slow occurrences of the transaction along with the Event ID, User, Operations Duration, Total Transaction Duration, and Timestamp of the event. Click on the Event ID or the Trace ID to open the [Trace View](/concepts/key-terms/tracing/trace-view/#product-walkthrough-trace-view-page). Other sets of transactions you can browse through are below.
```
Looks like they both go to the same page now

File: docs/product/performance/web-vitals/index.mdx [hunk 6]
```diff
@@ -60,7 +60,7 @@ The **Page Overview** page displays a "Page Loads" chart in the right sidebar of
 
 ### Samples List
 
-At the center of the **Page Overview**, Web Vital p75 values and scores are displayed. Clicking a Web Vital score will open a slideout panel containing a variety of **Page Load** samples with good to poor scores. Each sample contains an **Event ID** that can be clicked to open the [Event Detail](/product/sentry-basics/concepts/tracing/event-detail/) page for further investigation. If there is a [Replay](/product/session-replay/) or [Profile](/product/profiling/) associated with the sample page load, links will be included in the associated table columns.
+At the center of the **Page Overview**, Web Vital p75 values and scores are displayed. Clicking a Web Vital score will open a slideout panel containing a variety of **Page Load** samples with good to poor scores. Each sample contains an **Event ID** that can be clicked to open the [trace view](/concepts/key-terms/tracing/trace-view/#product-walkthrough-trace-view-page) page for further investigation. If there is a [Replay](/product/session-replay/) or [Profile](/product/profiling/) associated with the sample page load, links will be included in the associated table columns.
```
Comment (vivianyentran) [inline docs/product/performance/web-vitals/index.mdx:63]:
```suggestion
At the center of the **Page Overview**, Web Vital p75 values and scores are displayed. Clicking a Web Vital score will open a slideout panel containing a variety of **Page Load** samples with good to poor scores. Each sample contains an **Event ID** that can be clicked to open the [Trace View](/concepts/key-terms/tracing/trace-view/#product-walkthrough-trace-view-page) page for further investigation. If there is a [Replay](/product/session-replay/) or [Profile](/product/profiling/) associated with the sample page load, links will be included in the associated table columns.
```

We should be consistent with how we refer to this page & whether its capitalized or not

File: src/middleware.ts [hunk 7]
```diff
@@ -1115,11 +1115,11 @@ const REDIRECTS: Redirect[] = [
   },
   {
     from: '/product/performance/event-detail/',
-    to: '/product/sentry-basics/concepts/tracing/event-detail/',
+    to: '/product/sentry-basics/concepts/key-terms/tracing/trace-view/',
   },
   {
     from: '/product/sentry-basics/tracing/event-detail/',
-    to: '/product/sentry-basics/concepts/tracing/event-detail/',
+    to: '/product/sentry-basics/concepts/key-terms/tracing/trace-view/',
```
Comment (vivianyentran) [inline src/middleware.ts:1122]: I think you also need to add a redirect from '/product/sentry-basics/concepts/tracing/event-detail/' to the trace view page.

**Review by vivianyentran — APPROVED** [review]

File: docs/concepts/key-terms/tracing/distributed-tracing.mdx (hunk 2 above)
Comment (lizokm) [inline docs/concepts/key-terms/tracing/distributed-tracing.mdx:246]:
```suggestion
Through [Performance](/product/performance/) and [Discover](/product/discover-queries/), you can view trace data in the [trace details](/concepts/key-terms/tracing/trace-details/) page.
```

File: docs/concepts/key-terms/tracing/index.mdx [hunk 8]
```diff
@@ -6,7 +6,7 @@ description: "Learn more about the tracing features Sentry offers to help you tr
 
 [Distributed tracing](/product/sentry-basics/tracing/distributed-tracing/) provides a connected view of related errors and transactions by capturing interactions among your software systems. With tracing, Sentry tracks your software performance and displays the impact of errors across multiple systems. Tracing issues back through services connects your front-end to your back-end.
 
-The [Trace View](/product/sentry-basics/tracing/trace-view/) allows you to drill down into the details of a single trace and traverse every transaction in that trace. This accelerates your ability to debug slow services, identify related errors, and root out other bottlenecks. You can also drill all the way down into a span for a single transaction and information about it on the [Event Detail](/product/sentry-basics/tracing/event-detail/) page.
+The [Trace View](/product/sentry-basics/tracing/trace-view/) allows you to drill down into the details of a single trace and traverse every transaction in that trace. This accelerates your ability to debug slow services, identify related errors, and root out other bottlenecks.
```
Comment (lizokm) [inline docs/concepts/key-terms/tracing/index.mdx:9]:
```suggestion
The [Trace Details](/product/sentry-basics/tracing/trace-details/) allows you to drill down into the details of a single trace and traverse every transaction in that trace. This accelerates your ability to debug slow services, identify related errors, and root out other bottlenecks.
```

File: docs/platforms/java/guides/spring/advanced-usage.mdx [hunk 9]
```diff
@@ -8,7 +8,7 @@ Record user information from an HTTP request or by registering a Spring bean for
 
 ## Recording User Information From HTTP Request
 
-To record the user's IP address and `Principal#name` as the username so you can then view in [Event Details](/product/performance/event-detail/):
+To record the user's IP address and `Principal#name` as the username so you can then view in [trace view](/concepts/key-terms/tracing/trace-view/):
```
Comment (lizokm) [inline docs/platforms/java/guides/spring/advanced-usage.mdx:11]:
```suggestion
To record the user's IP address and `Principal#name` as the username so you can then view in [trace details view](/concepts/key-terms/tracing/trace-details/):
```

File: docs/product/performance/mobile-vitals/screen-loads.mdx [hunk 10]
```diff
@@ -88,4 +88,4 @@ Sentry automatically identifies sample events to help you investigate performanc
 
 You can use the sample list to drill down to and compare fast, average, and slow events of interest within a given screen.
 
-Clicking on a sample event will take you into either the query's span details within the span waterfall of the **Event Details** page or the profile icon to see the [flamegraph](/product/profiling/flame-charts-graphs/) for the event. To get different event samples, click the "Try Different Samples" button.
+Clicking on a sample event will take you into either the query's span details within the span waterfall of the [Trace view page](/concepts/key-terms/tracing/trace-view/#product-walkthrough-trace-view-page) or the profile icon to see the [flamegraph](/product/profiling/flame-charts-graphs/) for the event. To get different event samples, click the "Try Different Samples" button.
```
Comment (lizokm) [inline docs/product/performance/mobile-vitals/screen-loads.mdx:91]:
```suggestion
Clicking on a sample event will take you into either the query's span details within the span waterfall of the [Trace Details page](/concepts/key-terms/tracing/trace-details/) or the profile icon to see the [flamegraph](/product/profiling/flame-charts-graphs/) for the event. To get different event samples, click the "Try Different Samples" button.
```

File: docs/product/performance/queries.mdx [hunk 11]
```diff
@@ -117,7 +117,7 @@ These same events are also represented as triangles shown in the average duratio
 
 You can generate a new list of random sample events by clicking the "Try Different Samples" button at the bottom.
 
-From the sample list, you can drill down to specific good, average, or bad examples of a given query within a given endpoint. Click on an event ID to drill into the query's span details within the span waterfall of the **Event Details** page.
+From the sample list, you can drill down to specific good, average, or bad examples of a given query within a given endpoint. Click on an event ID to drill into the query's span details within the span waterfall of the [Trace View page](/concepts/key-terms/tracing/trace-view/#product-walkthrough-trace-view-page).
```
Comment (lizokm) [inline docs/product/performance/queries.mdx:120]:
```suggestion
From the sample list, you can drill down to specific good, average, or bad examples of a given query within a given endpoint. Click on an event ID to drill into the query's span details within the span waterfall of the [Trace Details page](/concepts/key-terms/tracing/trace-details/).
```

File: src/middleware.ts (hunk 7 above)
Comment (lizokm) [reply src/middleware.ts:1122]: Great call!

File: docs/concepts/key-terms/tracing/distributed-tracing.mdx [hunk 12]
```diff
@@ -243,4 +243,4 @@ Put simply: as a result of this head-based approach, where the decision is made
 
 ## Viewing Trace Data
 
-Through [Performance](/product/performance/) and [Discover](/product/discover-queries/), you can view trace data in the [event details](/product/sentry-basics/concepts/tracing/event-detail/).
+Through [Performance](/product/performance/) and [Discover](/product/discover-queries/), you can view trace data in the [Trace Details](/concepts/key-terms/tracing/trace-details/) page.
```
Comment (lizokm) [inline docs/concepts/key-terms/tracing/distributed-tracing.mdx:246]:
```suggestion
Through [Performance](/product/performance/) and [Discover](/product/discover-queries/), you can view trace data in the [Trace Details](/concepts/key-terms/tracing/trace-view/) page.
```

File: docs/concepts/key-terms/tracing/index.mdx [hunk 13]
```diff
@@ -6,7 +6,7 @@ description: "Learn more about the tracing features Sentry offers to help you tr
 
 [Distributed tracing](/product/sentry-basics/tracing/distributed-tracing/) provides a connected view of related errors and transactions by capturing interactions among your software systems. With tracing, Sentry tracks your software performance and displays the impact of errors across multiple systems. Tracing issues back through services connects your front-end to your back-end.
 
-The [Trace View](/product/sentry-basics/tracing/trace-view/) allows you to drill down into the details of a single trace and traverse every transaction in that trace. This accelerates your ability to debug slow services, identify related errors, and root out other bottlenecks. You can also drill all the way down into a span for a single transaction and information about it on the [Event Detail](/product/sentry-basics/tracing/event-detail/) page.
+The [Trace Details](/product/sentry-basics/tracing/trace-details/) allows you to drill down into the details of a single trace and traverse every transaction in that trace. This accelerates your ability to debug slow services, identify related errors, and root out other bottlenecks.
```
Comment (lizokm) [inline docs/concepts/key-terms/tracing/index.mdx:9]:
```suggestion
The [Trace Details](/product/sentry-basics/tracing/trace-view/) allows you to drill down into the details of a single trace and traverse every transaction in that trace. This accelerates your ability to debug slow services, identify related errors, and root out other bottlenecks.
```

File: docs/platforms/java/guides/spring/advanced-usage.mdx [hunk 14]
```diff
@@ -8,7 +8,7 @@ Record user information from an HTTP request or by registering a Spring bean for
 
 ## Recording User Information From HTTP Request
 
-To record the user's IP address and `Principal#name` as the username so you can then view in [Event Details](/product/performance/event-detail/):
+To record the user's IP address and `Principal#name` as the username so you can then view in [trace details view](/concepts/key-terms/tracing/trace-details/):
```
Comment (lizokm) [inline docs/platforms/java/guides/spring/advanced-usage.mdx:11]:
```suggestion
To record the user's IP address and `Principal#name` as the username so you can then view in [trace details view](/concepts/key-terms/tracing/trace-view/):
```

File: docs/product/performance/mobile-vitals/screen-loads.mdx [hunk 15]
```diff
@@ -88,4 +88,4 @@ Sentry automatically identifies sample events to help you investigate performanc
 
 You can use the sample list to drill down to and compare fast, average, and slow events of interest within a given screen.
 
-Clicking on a sample event will take you into either the query's span details within the span waterfall of the **Event Details** page or the profile icon to see the [flamegraph](/product/profiling/flame-charts-graphs/) for the event. To get different event samples, click the "Try Different Samples" button.
+Clicking on a sample event will take you into either the query's span details within the span waterfall of the [Trace Details page](/concepts/key-terms/tracing/trace-details/) or the profile icon to see the [flamegraph](/product/profiling/flame-charts-graphs/) for the event. To get different event samples, click the "Try Different Samples" button.
```
Comment (lizokm) [inline docs/product/performance/mobile-vitals/screen-loads.mdx:91]:
```suggestion
Clicking on a sample event will take you into either the query's span details within the span waterfall of the [Trace Details page](/concepts/key-terms/tracing/trace-view/) or the profile icon to see the [flamegraph](/product/profiling/flame-charts-graphs/) for the event. To get different event samples, click the "Try Different Samples" button.
```

File: docs/product/performance/queries.mdx [hunk 16]
```diff
@@ -117,7 +117,7 @@ These same events are also represented as triangles shown in the average duratio
 
 You can generate a new list of random sample events by clicking the "Try Different Samples" button at the bottom.
 
-From the sample list, you can drill down to specific good, average, or bad examples of a given query within a given endpoint. Click on an event ID to drill into the query's span details within the span waterfall of the **Event Details** page.
+From the sample list, you can drill down to specific good, average, or bad examples of a given query within a given endpoint. Click on an event ID to drill into the query's span details within the span waterfall of the [Trace Details page](/concepts/key-terms/tracing/trace-details/).
```
Comment (lizokm) [inline docs/product/performance/queries.mdx:120]:
```suggestion
From the sample list, you can drill down to specific good, average, or bad examples of a given query within a given endpoint. Click on an event ID to drill into the query's span details within the span waterfall of the [Trace Details page](/concepts/key-terms/tracing/trace-view/).
```

File: docs/product/performance/resources.mdx [hunk 17]
```diff
@@ -164,6 +164,6 @@ These same events are also represented as triangles shown in the average duratio
 
 You can generate a new list of random sample events by clicking the "Try Different Samples" button at the bottom.
 
-From the sample list, you can drill down to specific good, average, or bad examples of a given resource within a given page. Click on an event ID to drill into the resource's span details within the span waterfall of the **Event Details** page.
+From the sample list, you can drill down to specific good, average, or bad examples of a given resource within a given page. Click on an event ID to drill into the resource's span details within the span waterfall of the [Trace Details](/concepts/key-terms/tracing/trace-details/#product-walkthrough-trace-details-page) page.
```
Comment (lizokm) [inline docs/product/performance/resources.mdx:167]:
```suggestion
From the sample list, you can drill down to specific good, average, or bad examples of a given resource within a given page. Click on an event ID to drill into the resource's span details within the span waterfall of the [Trace Details](/concepts/key-terms/tracing/trace-view/#product-walkthrough-trace-details-page) page.
```

File: docs/product/performance/transaction-summary.mdx [hunk 18]
```diff
@@ -44,7 +44,7 @@ While you can use the search field to narrow down your events, you can also filt
 
 ### Transaction Table
 
-On initial load, the table displays slow occurrences of the transaction along with the Event ID, User, [Operations Breakdown](/product/sentry-basics/tracing/event-detail/#operations-breakdown), Total Transaction Duration, and Timestamp of the event. Click on the Event ID to open the [event detail](/product/sentry-basics/tracing/event-detail/) or the Trace ID to view the [full trace](/product/sentry-basics/tracing/trace-view). Other sets of transactions you can browse through are below.
+On initial load, the table displays slow occurrences of the transaction along with the Event ID, User, Operations Duration, Total Transaction Duration, and Timestamp of the event. Click on the Event ID or the Trace ID to open the [Trace Details](/concepts/key-terms/tracing/trace-details/#product-walkthrough-trace-details-page) page. Other sets of transactions you can browse through are below.
```
Comment (lizokm) [inline docs/product/performance/transaction-summary.mdx:47]:
```suggestion
On initial load, the table displays slow occurrences of the transaction along with the Event ID, User, Operations Duration, Total Transaction Duration, and Timestamp of the event. Click on the Event ID or the Trace ID to open the [Trace Details](/concepts/key-terms/tracing/trace-view/#product-walkthrough-trace-details-page) page. Other sets of transactions you can browse through are below.
```

File: docs/product/performance/web-vitals/index.mdx [hunk 19]
```diff
@@ -60,7 +60,7 @@ The **Page Overview** page displays a "Page Loads" chart in the right sidebar of
 
 ### Samples List
 
-At the center of the **Page Overview**, Web Vital p75 values and scores are displayed. Clicking a Web Vital score will open a slideout panel containing a variety of **Page Load** samples with good to poor scores. Each sample contains an **Event ID** that can be clicked to open the [Event Detail](/product/sentry-basics/concepts/tracing/event-detail/) page for further investigation. If there is a [Replay](/product/session-replay/) or [Profile](/product/profiling/) associated with the sample page load, links will be included in the associated table columns.
+At the center of the **Page Overview**, Web Vital p75 values and scores are displayed. Clicking a Web Vital score will open a slideout panel containing a variety of **Page Load** samples with good to poor scores. Each sample contains an **Event ID** that can be clicked to open the [Trace Details](/concepts/key-terms/tracing/trace-details/#product-walkthrough-trace-details-page) page for further investigation. If there is a [Replay](/product/session-replay/) or [Profile](/product/profiling/) associated with the sample page load, links will be included in the associated table columns.
```
Comment (lizokm) [inline docs/product/performance/web-vitals/index.mdx:63]:
```suggestion
At the center of the **Page Overview**, Web Vital p75 values and scores are displayed. Clicking a Web Vital score will open a slideout panel containing a variety of **Page Load** samples with good to poor scores. Each sample contains an **Event ID** that can be clicked to open the [Trace Details](/concepts/key-terms/tracing/trace-view/#product-walkthrough-trace-details-page) page for further investigation. If there is a [Replay](/product/session-replay/) or [Profile](/product/profiling/) associated with the sample page load, links will be included in the associated table columns.
```

**Review by JonasBa — APPROVED** [review]

**Pull request merged**